tinyagentos/mcp/supervisor.py

```
@@ -131,11 +131,16 @@ async def uninstall(self, server_id: str) -> dict:
-            try:
-                await self.stop(sid)
-            except Exception:
-                logger.exception("mcp stop_all: error stopping %s", sid)
+        sids = list(self._processes.keys())
+        if not sids:
+            return
+        results = await asyncio.gather(
+            *(self.stop(sid) for sid in sids),
+            return_exceptions=True,
+        )
+        for sid, result in zip(sids, results):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)